### PR TITLE
Add skew function to css3-generator utils

### PR DESCRIPTION
**Pull request by PascalPiche**

### CODE REVIEW

File: src/utils/css3-generator.js [hunk 1]
```diff
@@ -47,16 +47,41 @@
 		return 'rotate' + prefix + theta + suffix;
 	};
 	
+	var getSkew = function (x, y, unit) {
+		if (!!!unit) {
+			unit = 'deg';
+		}
+
+		if (unit != 'deg' && unit != 'rad' && unit != 'turn') {
```
Comment (fhamon) [inline src/utils/css3-generator.js]: We should do a strict comparison `!==` 😁

File: src/utils/css3-generator.js [hunk 2]
```diff
@@ -47,16 +47,41 @@
 		return 'rotate' + prefix + theta + suffix;
 	};
 	
+	var getSkew = function (x, y, unit) {
+		if (!!!unit) {
```
Comment (fhamon) [inline src/utils/css3-generator.js]: Maybe we should only have one `!`, just to make the code clearer to read 🤸‍♂

File: src/utils/css3-generator.js (hunk 1 above)
Comment (PascalPiche) [reply src/utils/css3-generator.js]: fixer

File: src/utils/css3-generator.js (hunk 2 above)
Comment (PascalPiche) [reply src/utils/css3-generator.js]: fixer

**Pull request merged**